### DISCUSSION TITLE
Install local packages

### DISCUSSION THREAD

**Discussion by ispringle** [espanso/espanso#1181] [Feature requests and Ideas]
It'd be nice if `espanso install` also allowed for installing a local package, similar to how `pip` lets you do `pip install ./`. Additionally it'd be nice to install a package from a remote origin such like `espanso install github.com/user/espanso-package`

Both would come in handy for testing packages.

**Comment by federico-terzi** [comment]
Hey @pard68,

Thank you for the feedback! I'll add the local installation to the roadmap :)

Regarding the URL installation, you can already do that since the last version of espanso, please take a look here: https://github.com/federico-terzi/espanso/releases/tag/v0.6.0

Cheers :)
